src/config/build_config.py

```diff
@@ -4,7 +4,7 @@
 # Frameworks for which you want to disable both builds and tests
 DISABLE_FRAMEWORK_TESTS = []
```

Comment:
You can include MXNet and Pytorch in the list to disable their build and test.

Reply:
Should be Okay to test them all together